xml/rmt_install.xml

```diff
@@ -35,11 +35,9 @@
 
   <para>
    Downloaded packages are stored in<filename>
-   /usr/share/rmt/public/repo</filename>, which is a symbolic link to
-   <filename>/var/lib/rmt/public/repo/SUSE/Products/</filename>. (You may
-   change this location by changing the symbolic link, and then updating the &rmt;
-   configuration files in <filename>/etc/nginx/vhosts.d/</filename> with the
-   new symbolic link.)
+    /usr/share/rmt/public/repo</filename>, which is a symbolic link to
+   <filename>/var/lib/rmt/public/repo/</filename>. You can change this location
+   by purely changing the symbolic link.
```

Comment:
I suggest omitting the sentence "You can change this location..." It raises questions, and getting derailed into talking about symlinks seems like a good thing to avoid.